server/plugin/webhook.go

```diff
@@ -385,18 +390,22 @@ func (p *Plugin) postPullRequestEvent(event *github.PullRequestEvent) {
 
 	for _, sub := range subs {
 		if !sub.Pulls() && !sub.PullsMerged() && !sub.PullsCreated() {
```

Comment:
After the loop we can log whether `foundDocsRepoSub` is `true`

```suggestion
	foundDocsRepoSub := false
	for _, sub := range subs {
		if sub.Repository == "mattermost/docs" {
			foundDocsRepoSub = true
			b, err := json.Marshal(sub)
			if err != nil {
				p.API.LogError("error marshaling subscription", "error", err.Error())
			} else {
				p.API.LogDebug("logging docs subscription, "sub", string(b))
			}
		}

		if !sub.Pulls() && !sub.PullsMerged() && !sub.PullsCreated() {
```